react-app/src/components/property/PropertyForms.tsx

```diff
@@ -193,7 +193,7 @@ export const GeneralInformationForm = (props: IGeneralInformationForm) => {
               validate: (val, formVals) =>
                 (val.length <= 9 &&
                   (val.length > 0 || formVals['PIN'].length > 0 || propertyType === 'Building')) ||
-                'Must have set either PID or PIN',
+                'Must have set either PID or PIN not exceeding 9 digits.',
```

Comment:
Tried doing this validation after converting val to a string. It will go through like that, but then the value saved in the database is NULL, so something is getting lost along the way.

Reply:
I've corrected this to account for the new behavior where TextFormField will parseFloat onBlur. Something to note for the future though: doing that parseFloat means that entering enough digits will now lock up the text field since the parsed float will be in scientific notation.